### PR TITLE
Implement command-line arguments for host, port and share

### PR DESCRIPTION
**Pull request by Subhanshu0027**

### CODE REVIEW

**Comment by ashish-aesthisia** [comment]
@Subhanshu0027 Please update the documentation for --share.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ To use LLMinator, follow these simple steps:
 |----------|----------|----------|
 | --host | 127.0.0.1 | Host or IP address on which the server will listen for incoming connections.|
 | --port| 7860 | Launch gradio with given server port |
+| -- share| False | This generates a public shareable link that you can send to anybody|
```
Comment (ashish-aesthisia) [inline README.md:34]: Extra space between -- & share

**Review by ashish-aesthisia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Subhanshu0027) [reply README.md]: Removed the spaces

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ To use LLMinator, follow these simple steps:
 |----------|----------|----------|
 | --host | 127.0.0.1 | Host or IP address on which the server will listen for incoming connections.|
```
Comment (ashish-aesthisia) [inline README.md:32]: Consistency issue, sometime there is . available & somewhere not. 
Space is also missing sometimes. 

Please make sure to keep the consistency across you code.

**Review by ashish-aesthisia — CHANGES_REQUESTED** [review]

**Pull request merged**